modules/media/php/NDB_Menu_Filter_media.class.inc

```diff
@@ -82,20 +100,29 @@ class NDB_Menu_Filter_Media extends NDB_Menu_Filter
             $siteList = array('' => 'All User Sites') + $siteList;
         }
 
-        $instrumentList   = [null => 'Any'];
+        $instrumentList   = [];
         $instrumentsQuery = $db->pselect(
             "SELECT Test_name, Full_name FROM test_names ORDER BY Test_name",
             []
         );
         foreach ($instrumentsQuery as $instrument) {
-            $instrumentList[$instrument['Test_name']] = $instrument['Full_name'];
+            $instrumentList[$instrument['Full_name']] = $instrument['Full_name'];
+        }
+
+        $fileTypeList  = [];
+        $fileTypeQuery = $db->pselect("SELECT file_type FROM media", []);
+        foreach ($fileTypeQuery as $filetype) {
+            $fileTypeList[$filetype['file_type']] = $filetype['file_type'];
         }
 
         // Form Elements
-        $this->addBasicText('pscid', 'PSCID:', ["size" => 9, "maxlength" => 7]);
-        $this->addSelect('instrument', 'Instrument:', $instrumentList);
-        $this->addSelect('visit_label', 'Visit Label:', $visitList);
-        $this->addSelect('for_site', 'For Site:', $siteList);
+        $this->addBasicText('pSCID', 'PSCID', ["size" => 9, "maxlength" => 7]);
+        $this->addBasicText('fileName', 'File Name');
+        $this->addSelect('fileType', 'File type', $fileTypeList);
+        $this->addSelect('visitLabel', 'Visit Label', $visitList);
+        $this->addSelect('site', 'For Site', $siteList);
+        $this->addBasicText('uploadedBy', 'Uploaded By');
+        $this->addSelect('instrument', 'Instrument', $instrumentList);
```

Comment:
This is all rendered via React isn't it? Do we need to keep these calls to LorisForm around?
Is it a round-about way of providing the data needed for the AJAX request?